src/components/PlanList.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import api from "./Api";
+import { Link } from "react-router-dom";
+import { plansApiUrl } from "./constants";
+
+export default class PlanList extends React.Component {
+  state = {
+    plans: [],
+  };
+
+  setPlans = (data) => {
+    this.setState({ plans: data.plans });
+  };
+
+  componentDidMount() {
+    api(plansApiUrl, this.setPlans);
+  }
+
+  render() {
+    let { plans } = this.state;
+    return (
+      <div>
+        <h1 className="text-center"> Plans </h1>
+        <table className="table table-hover text-center">
+          <thead>
+            <tr>
+              <th scope="col"> Name </th> <th scope="col"> Fee </th>
+              <th colSpan="1"> </th>
+            </tr>
+          </thead>
+          <tbody>
+            {plans.map((plan) => (
```

Comment:
Add space between `{` and `plans`

Reply:
ma space insert krti hun but save krny pa phir sy chali jati